app/templates/default/exp.html

```diff
@@ -570,7 +570,21 @@ <h5 class="modal-title" id="exampleModalLabel">SupportService Survey</h5>
 
     // JQuery to populate NPS buttons
     $(document).ready(function() {
-                
+        // Use bootstrapping method to provide immediate values to javascript client
```

Comment:
Can we continue to try to do this with external JS instead of inline?

Reply:
can you help me with how to handle the external JS with webpack? That part is confusing to me, I haven't done it before. Unless by external you mean a separate JS file and still just include that in the webpack bundle?

Reply:
We are not using webpack at all (afaik). I am just saying put the code into an external JS file and load that instead of inline JS.